lms/djangoapps/teams/static/teams/js/views/topic_card.js

```diff
@@ -46,7 +46,9 @@
                         gettext('View Teams in the %(topic_name)s Topic'),
                         { topic_name: this.model.get('name') }, true
                     ));
-                    return '<span class="sr">' + screenReaderText + '</span><i class="icon fa fa-arrow-right" aria-hidden="true"></i>';
+                    return '<span class="sr">' +
```

Comment:
OK, as long as quality ends up passing, let's touch do the safe templating work here now. This PR is already very large and prone to get merge conflicts.